CONTRIBUTING.md

```diff
@@ -381,6 +381,68 @@ It offers a number of functionalities that improve development:
   relative import from the file system. As a consequence, the mechanism for
   overriding import paths for tests and Storybook's pages will not work.
 
+## Unit tests
+
+The project has unit tests for both C++ and QML code.
+
+### C++ unit tests
+
+C++ unit tests are localized under `ui/StatusQ/tests`. They can be executed
+easily from QtCreator. Just open the `StatusQ` project in `QtCreator`
+(`status-desktop/ui/StatusQ/CMakeLists.txt`) and select the appropriate target (each test is a separate target).
```

Comment:
```suggestion
C++ unit tests are localized under `ui/StatusQ/tests`. They can be executed
easily from QtCreator or make target:

- Open the `StatusQ` project in `QtCreator` (`status-desktop/ui/StatusQ/CMakeLists.txt`) and select the appropriate target (each test is a separate target).

- Run the `run-statusq-tests` target. E.g. `make -j10 run-statusq-tests ARGS="-R ModelEntryTest"`

```

Reply:
not sure QtCreator is a popular IDE around contributors.

Reply:
I'm not sure either. But it's good to add at least some hints for QtCreator which is used commonly by CCs.

Reply:
That's what I was trying to point out. I'm not sure it's commonly used by CCs 😄 
AFAIK most people use it only when they don't know of any other alternative to accomplish something (like running tests).